home/.config/nvim/lua/plugins/telescope.lua

```diff
@@ -0,0 +1,32 @@
+return {
+  "nvim-telescope/telescope.nvim",
+  dependencies = {
+    "nvim-lua/plenary.nvim",
+
+    "kkharji/sqlite.lua",
+    "folke/todo-comments.nvim",
+    "BurntSushi/ripgrep",
+  },
+  config = function()
+    local telescope = require("telescope")
+
+    telescope.setup({
+      defaults = {
+        path_display = { "smart" },
+      },
+    })
+
+    local keymap = vim.keymap -- for conciseness
+
+    keymap.set("n", "<C-p>", ":Telescope git_files<CR>", { desc = "Files: Known to git" })
+    keymap.set("n", "<leader>b", ":Telescope buffers<CR>", { desc = "Buffers: Open buffers" })
+    keymap.set("n", "<leader>bf", ":Telescope current_buffer_fuzzy_find<CR>", { desc = "Buffers: Find in current" })
+    keymap.set("n", "<leader>bv", ":Telescope treesitter<CR>", { desc = "Buffer: Variables" })
+    keymap.set("n", "<leader>c", ":Telescope git_commits<CR>", { desc = "Git: Commits" })
+    keymap.set("n", "<leader>ft", "<cmd>TodoTelescope<cr>", { desc = "Find todos" })
+    keymap.set("n", "<leader>gg", ":Telescope live_grep<CR>", { desc = "Files: Search in files" })
+    keymap.set("n", "g<C-p>", ":Telescope find_files<CR>", { desc = "Files: All files in directory" })
+    keymap.set("n", "gs<C-p>", ":Telescope git_status<CR>", { desc = "Files: Modified (git status)" })
+    keymap.set("n", "tr", ":Telescope resume<CR>", { desc = "Telescope: resume last" })
```

Comment:
No way. This has to be at least a leader command. I use `t` all the time.

Reply:
I'm good with removing this. I think I added it back when I was trying to find a way to easily get back to a list of search results.